src/Game.vue

```diff
@@ -4,8 +4,8 @@ import { allWords } from './words'
 import Keyboard from './Keyboard.vue'
 import { LetterState } from './types'
 
-// try to guess this by actually playing the game :)
-const answer = atob('aGVsbG8=')
+// Pick a random word from dictonary
+const answer = allWords[Math.floor(Math.random() * allWords.length)]
```

Comment:
I think this is an Easter egg from Evan. I mean if you run `atob('aGVsbG8=')` it returns `hello`... and try typing hello in the game. `aGVsbG8=` is base64 encoded `hello`. I don't see the need of changing this.